### PR TITLE
Named configuration updates

### PR DESCRIPTION
**Pull request by carabasdaniel**
Depends on https://github.com/aserto-dev/topaz/pull/255

### CODE REVIEW

File: docs/command-line-interface/topaz-cli/configuration.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ sidebar_position: 2
 ---
 # Configuration
 
-Topaz requires a configuration file to run. By default, the location of the config file is `$HOME/.config/topaz/cfg/config.yaml`. 
+Topaz requires a configuration file to run. By default, the location of the config file is `$XDG_CONFIG_HOME/topaz/cfg/config.yaml`.
```
Comment (ronenh) [inline docs/command-line-interface/topaz-cli/configuration.mdx:6]: XDG_CONFIG_HOME isn't defined in most people's shell. What do we use if it's not there?
Perhaps this sentence should describe the precedence order.

File: docs/command-line-interface/topaz-cli/configuration.mdx [hunk 2]
```diff
@@ -14,13 +14,15 @@ The [templates install](templates#installing-a-template) command automatically c
 ```
 Flags:
   -h, --help                         Show context-sensitive help.
-      --no-check                     disable local container running check ($TOPAZ_NO_CHECK)
+  -N, --no-check                     disable local container status check ($TOPAZ_NO_CHECK)
```
Comment (ronenh) [inline docs/command-line-interface/topaz-cli/configuration.mdx:17]: I always found this option's description a bit vague. Maybe we can take this opportunity and rephrase it.
Something like "execute command without verifying that a local topaz container is running".

File: docs/command-line-interface/topaz-cli/configuration.mdx [hunk 3]
```diff
@@ -14,13 +14,15 @@ The [templates install](templates#installing-a-template) command automatically c
...
 
   -n, --policy-name=STRING           policy name
   -l, --local-policy-image=STRING    local policy image name
   -r, --resource=STRING              resource url
   -p, --stdout                       generated configuration is printed to stdout but not saved
   -d, --edge-directory               enable edge directory
+  -f, --force                        forcefully create configuration
```
Comment (ronenh) [inline docs/command-line-interface/topaz-cli/configuration.mdx:24]: Maybe rephrase to "skip confirmation prompt".

File: docs/command-line-interface/topaz-cli/configuration.mdx [hunk 4]
```diff
@@ -14,13 +14,15 @@ The [templates install](templates#installing-a-template) command automatically c
...
+  -c, --config="config.yaml"         use topaz configuration file
```
Comment (ronenh) [inline docs/command-line-interface/topaz-cli/configuration.mdx:25]: I'd just say "topaz configuration file" (without "use"). Similar to the other options like `-n`, `-r`, etc.

File: docs/command-line-interface/topaz-cli/configuration.mdx [hunk 5]
```diff
@@ -14,13 +14,15 @@ The [templates install](templates#installing-a-template) command automatically c
 ```
 Flags:
   -h, --help                         Show context-sensitive help.
-      --no-check                     disable local container running check ($TOPAZ_NO_CHECK)
+  -N, --no-check                     disable local container status check ($TOPAZ_NO_CHECK)
 
   -n, --policy-name=STRING           policy name
   -l, --local-policy-image=STRING    local policy image name
   -r, --resource=STRING              resource url
   -p, --stdout                       generated configuration is printed to stdout but not saved
```
Comment (ronenh) [inline docs/command-line-interface/topaz-cli/configuration.mdx:22]: "print to stdout instead of updating the configuration file"

File: docs/command-line-interface/topaz-cli/configuration.mdx [hunk 6]
```diff
@@ -37,6 +39,18 @@ To generate a configuration file for a local OCI image, use the following comman
...
 
+## Named configurations
+
+Topaz CLI allows the user to have multiple configuration files and easily switch between them when running the topaz container. 
+
+When using templates the generated configuration file uses the name of the template (ex. for `todo` template it will generate a `$XDG_CONFIG_HOME/topaz/cfg/todo.yaml`).
+
+To see available configuration files you can use the `topaz list` command, this will also highlight the current active configuration file. 
+
+If you want to start a different configuration file, you can just specify the file you want to use in the `topaz start/run --config=<name_of_file.yaml>` command. This will automatically switch you active configuration file to the one you used.
```
Comment (ronenh) [inline docs/command-line-interface/topaz-cli/configuration.mdx:50]: "If you want to start topaz with a different configuration file, ..."

File: docs/command-line-interface/topaz-cli/configuration.mdx [hunk 7]
```diff
@@ -37,6 +39,18 @@ To generate a configuration file for a local OCI image, use the following comman
...
+
+Note: This change affect the topaz container name. This changes from the default `topaz` to the `topaz-<configname>` format.
```
Comment (ronenh) [inline docs/command-line-interface/topaz-cli/configuration.mdx:52]: What exactly do you mean by "This change"? Do you mean that the container name is `topaz` by default, but if you run `topaz start -c myfile.yaml` then it will be `topaz-myfile`? Or is it _always_ `topaz-<configname>`?

Comment (carabasdaniel) [reply docs/command-line-interface/topaz-cli/configuration.mdx:52]: The container will always be in the format `topaz-<configname>` once the named configuration changes are merged.

**Review by gertd — APPROVED** [review]
